kura/examples/org.eclipse.kura.example.gpio.led/src/org/eclipse/kura/example/gpio/led/LedExample.java

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Eurotech and/or its affiliates
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech
+ *******************************************************************************/
+
+package org.eclipse.kura.example.gpio.led;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.kura.configuration.ConfigurableComponent;
+import org.eclipse.kura.gpio.GPIOService;
+import org.eclipse.kura.gpio.KuraClosedDeviceException;
+import org.eclipse.kura.gpio.KuraGPIODeviceException;
+import org.eclipse.kura.gpio.KuraGPIOPin;
+import org.eclipse.kura.gpio.KuraUnavailableDeviceException;
+import org.osgi.service.component.ComponentContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LedExample implements ConfigurableComponent {
+
+    private static final Logger s_logger = LoggerFactory.getLogger(LedExample.class);
+    private static final String APP_ID = "org.eclipse.kura.example.gpio.led.LedExample";
+   
+
+    private GPIOService myservice;
+    private LedOptions options;
+
+    protected synchronized void bindGPIOService(final GPIOService gpioService) {
+        this.myservice = gpioService;
+    }
+
+    protected synchronized void unbindGPIOService(final GPIOService gpioService) {
+        this.myservice = null;
+    }
+
+    protected void activate(ComponentContext componentContext, Map<String, Object> properties) {
+        s_logger.info("Bundle " + APP_ID + " has started with config!");
```

Comment:
To use better the features of log4j, it's better to change this as:
`logger.info("Bundle {} has started with config!", APP_ID);`

Reply:
Please do the same also with the deactivate method.